src/pages/workspaces/workspace/workflows/WorkflowView.js

```diff
@@ -562,13 +585,14 @@ const WorkflowView = _.flow(
       selectedEntityType, entityMetadata, entitySelectionModel, versionIds = [], useCallCache, currentSnapRedacted, savedSnapRedacted, wdl
     } = this.state
     const { name, methodRepoMethod: { methodPath, methodVersion, methodNamespace, methodName, sourceRepo }, rootEntityType } = modifiedConfig
+    const entityTypes = _.keys(entityMetadata)
+    const possibleSetTypes = findPossibleSets(entityTypes)
     const modified = !_.isEqual(modifiedConfig, savedConfig)
     const noLaunchReason = Utils.cond(
       [saving || modified, () => 'Save or cancel to Launch Analysis'],
       [!_.isEmpty(errors.inputs) || !_.isEmpty(errors.outputs), () => 'At least one required attribute is missing or invalid'],
-      [this.isMultiple() && !entityMetadata[rootEntityType], () => `There are no ${selectedEntityType}s in this workspace.`],
```

Comment:
we deleted this check for whether the selectedEntityType currently exists in the data model, since you can now select a set type that doesn't yet exist. is it okay to have this totally removed now?

Reply:
This needs to stay in. You could import a config that specifies an entity type that just straight up isn't in the workspace, so we need to handle that.

Reply:
Ok, adding back this: `[this.isMultiple() && (!entityMetadata[rootEntityType] || !_.includes(rootEntityType, possibleSetTypes)), () => `There are no ${selectedEntityType}s in this workspace.`],`